python/cudf/setup.py

```diff
@@ -1,5 +1,5 @@
-# Copyright (c) 2018-2021, NVIDIA CORPORATION.
-# modify on purpose :P
+# Copyright (c) 2018-2020, NVIDIA CORPORATION.
```

Comment:
No no please keep 2021. It should not pass for any year that is not 2022.

Reply:
I'm just testing in CI, it will not pass. Will change to prove for 2021 as well next

Reply:
Kudo for 2020:
```
FAILED: copyright check; end output
```

Reply:
Failed for 2020: https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/cudf/job/prb/job/cudf-style/23578/console

Reply:
Failed for 2021 too: https://gpuci.gpuopenanalytics.com/blue/organizations/jenkins/rapidsai%2Fgpuci%2Fcudf%2Fprb%2Fcudf-style/detail/cudf-style/23579/pipeline